### PR TITLE
feat: commit lint is added in the frontend

### LINKED ISSUE #1131: integrate commit lint in frontend
## Is your feature request related to a problem?

*Please describe.*
https://github.com/conventional-changelog/commitlint

## Describe the solution you'd like
Configure commit lint in frontend

### PR DESCRIPTION
**Pull request by palash-signoz**
Close #1131

### CODE REVIEW

File: frontend/.husky/commit-msg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+cd frontend && yarn commitlint --edit $1
```
Comment (pranshuchittora) [inline frontend/.husky/commit-msg:4]: Add newline

Comment (palash-signoz) [reply frontend/.husky/commit-msg:4]: didn't get this one

Comment (pranshuchittora) [reply frontend/.husky/commit-msg:4]: Add new line at the end of file _(with Axolo)_

Comment (palash-signoz) [reply frontend/.husky/commit-msg:4]: got it _(with Axolo)_

Comment (pranshuchittora) [inline frontend/.husky/commit-msg:4]: Let's use npm here. Why?
Because it will break if someone want to contribute and doesn't have yarn installed.

Comment (palash-signoz) [reply frontend/.husky/commit-msg]: there are other in pre-commit stage let's cover in another PR

**Review by pranshuchittora — APPROVED** [review]

**Pull request merged**